modules/claims_api/app/models/concerns/claims_api/file_data.rb

```diff
@@ -25,7 +25,7 @@ def description
       end
 
       def set_file_data!(file_data, doc_type, description = nil)
-        if Flipper.enabled? :claims_load_testing
+        if (Flipper.enabled? :claims_load_testing) || Settings.claims_api.pdf_generator_526.mock
```

Comment:
Wondering if it's confusing to have the setting that controls pdf generator mocking also control Amazon s3 mocking? Should we make the name more generic, or include s3 at the end?